src/pages/blog/rss.xml.ts

```diff
@@ -0,0 +1,28 @@
+import rss from "@astrojs/rss";
+import { getCollection } from "astro:content";
+import sanitizeHtml from "sanitize-html";
+import MarkdownIt from "markdown-it";
+import type { APIRoute } from "astro";
+const parser = new MarkdownIt();
+
+export const GET: APIRoute = async (context) => {
+  const blog = await getCollection("blog");
+  return rss({
+    title: "Grain Blog",
+    description: "News for the Grain Programming Language",
+    site: `${context.site}/blog`,
+    items: blog
+      .toSorted((x, y) => y.data.date.getTime() - x.data.date.getTime())
+      .map((post) => ({
+        link: `/blog/${post.id}/`,
+        title: post.data.title,
+        pubDate: post.data.date,
+        description: post.data.subtitle,
+        categories: post.data.tags ?? undefined,
+        author: post.data.author,
+        content: sanitizeHtml(parser.render(post.body), {
```

Comment:
Why are we sanitizing this html, we'll be manually reviewing it on GitHub won't we?

Reply:
Yeah I just copied this from the astro example, but I figured it wouldn't hurt